mne/io/nsx/nsx.py

```diff
@@ -0,0 +1,562 @@
+# Author: Proloy Das <pdas6@mgh.harvard.edu>
+#
+# License: BSD-3-Clause
+import os
+from datetime import datetime, timezone
+
+import numpy as np
+
+from ..constants import FIFF
+from ..meas_info import _empty_info
+from ..base import BaseRaw, _get_scaling
+from ..utils import _read_segments_file
+
+from ...annotations import Annotations
+from ...utils import logger, fill_doc, warn
+
+
+# common channel type names mapped to internal ch types
+# CH_TYPE_MAPPING = {
+#     "EEG": FIFF.FIFFV_EEG_CH,
+#     "SEEG": FIFF.FIFFV_SEEG_CH,
+#     "CC": FIFF.FIFFV_SEEG_CH,
+#     "ECOG": FIFF.FIFFV_ECOG_CH,
+#     "DBS": FIFF.FIFFV_DBS_CH,
+#     "EOG": FIFF.FIFFV_EOG_CH,
+#     "ECG": FIFF.FIFFV_ECG_CH,
+#     "EMG": FIFF.FIFFV_EMG_CH,
+#     "BIO": FIFF.FIFFV_BIO_CH,
+#     "RESP": FIFF.FIFFV_RESP_CH,
+#     "TEMP": FIFF.FIFFV_TEMPERATURE_CH,
+#     "MISC": FIFF.FIFFV_MISC_CH,
+#     "SAO2": FIFF.FIFFV_BIO_CH,
+# }
+
+CH_TYPE_MAPPING = {
+    "CC": "SEEG",
+}
+
+
+DATA_BYTE_SIZE = 2
```

Comment:
Could these be different in different nsx files? I don't know, just asking. If so, they would probably be better exposed as parameters or better yet inferred from the file itself. If not, a link here to the blackrock specs saying that that is the byte size etc. would be nice in a comment.